test/unit/collection.test.js

```diff
@@ -25,7 +25,8 @@ describe('Collection', function () {
                         script: {
                             id: 'my-script-1',
                             type: 'text/javascript',
-                            exec: ['console.log("This doesn\'t matter");']
+                            exec: ['console.log("This doesn\'t matter");'],
+                            packages: {}
```

Comment:
Why do we need to add the empty key everywhere? It's supposed to be optional, no?

Reply:
Yep yep, just needed to add this to the assertions after conversion since we're defaulting to an empty object, no need for this here.